### DISCUSSION TITLE
Lots of progress bar output

### DISCUSSION THREAD

**Discussion by vema1256** [Lightning-AI/pytorch-lightning#9530] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
PL Version: 1.4.2

Im running the example notebook on [text transformer ](https://github.com/PyTorchLightning/lightning-tutorials/blob/4fe3370eac9c448eceb36b835ff49ca30de7d404/lightning_examples/text-transformers/text-transformers.py) and the output is endless  progress bar output. Is there a way to suppress/control it.

100%|████████████████████| 4/4 [00:00<00:00, 39016.78it/s]
100%|████████████████████| 4/4 [00:00<00:00, 38304.15it/s]
100%|████████████████████| 4/4 [00:00<00:00, 38746.46it/s]
100%|████████████████████| 4/4 [00:00<00:00, 38568.31it/s]
100%|████████████████████| 4/4 [00:00<00:00, 38568.31it/s]
100%|████████████████████| 4/4 [00:00<00:00, 39107.73it/s]
100%|████████████████████| 4/4 [00:00<00:00, 37532.92it/s]
100%|████████████████████| 4/4 [00:00<00:00, 37532.92it/s]

**Comment by rohitgr7** [comment]
using pycharm?

**Reply by tchaton** [reply]
Hey @vema1256,

You can set `Trainer(progress_bar_refresh_rate=30)`

**Reply by vema1256** [reply]
> using pycharm?

@rohitgr7   Im running it on the terminal. Tested on both mac, ubuntu, and google colab environments too. Problem persists.

@tchaton . Ive tested with the progress_bar_refresh_rate too. This doesn't help either.

**Reply by vema1256** [reply]
@tchaton My previous conda environment was with python=3.8 I created another conda environment with python=3.6 and the problem seems to have resolved.
